src/app/TraitReflections.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Backpack\PageManager\app;
+
+trait TraitReflections
+{
+    public function checkForTemplatesAndUniquePagesNotDistinct()
+    {
+        if (config('backpack.pagemanager.page_model_class') != config('backpack.pagemanager.unique_page_model_class')) {
+            return;
+        }
+
+        $uniquePages = collect($this->getUniquePages())->pluck('name');
```

Comment:
Sigh. It looks like all these getUniquePages and getTemplates things should just return collections...